static.json

```diff
@@ -7,6 +7,9 @@
     },
     "/statusboard": {
       "origin": "https://ember-learn.github.io/statusboard/"
+    },
+    "/rfcs": {
+      "origin": "https://emberjs.github.io/rfcs/"
```

Comment:
This url seems to fail on my end, is that happening for you too?

Reply:
DNS for emberjs.github.io is stealing the URL.